features/user/managing_customers/toggling_customer_account.feature

```diff
@@ -24,3 +24,13 @@ Feature: Toggling a customer account
         And I save my changes
         Then I should be notified that it has been successfully edited
         And this customer should be disabled
+
+    @ui
+    Scenario: Verifying customer account
```

Comment:
IMO this scenario should be extracted to a separate file as it provides a different business value than enabling/disabling customers